apps/scov/main.c

```diff
@@ -32,6 +32,7 @@
 #include <ctype.h> /* for toupper */
 
 #define PRINT_ENABLED 1
+#define MAX_COV_SUBCRIPTIONS 100000
```

Comment:
you have this defined in the makefile but for it to use it this will need a check around it:

#ifndef MAX_COV_SUBCRIPTIONS
#define MAX_COV_SUBCRIPTIONS 100000
#endif